src/applications/claims-status/tests/utils/helpers.unit.spec.js

```diff
@@ -662,6 +687,42 @@ describe('Disability benefits helpers: ', () => {
     });
   });
 
+  describe('isClaimOpen', () => {
+    context('when status is COMPELTE', () => {
+      const status = 'COMPLETE';
+      context('when closeDate is null', () => {
+        it('should return false', () => {
+          const isOpen = isClaimOpen(status, null);
+          expect(isOpen).to.be.false;
+        });
+      });
+
+      context('when closeDate exists', () => {
+        it('should return false', () => {
+          const isOpen = isClaimOpen(status, '2024-01-01');
+          expect(isOpen).to.be.false;
+        });
+      });
+    });
+
+    context('when status is not COMPELTE', () => {
```

Comment:
typo here: "COMPELTE" should be "COMPLETE"

Reply:
updated thanks!